libraries/hotstuff/qc_chain.cpp

```diff
@@ -239,25 +226,25 @@ namespace eosio { namespace hotstuff {
       return b;
    }
 
-   bool qc_chain::evaluate_quorum(const extended_schedule & es, fc::unsigned_int finalizers, const fc::crypto::blslib::bls_signature & agg_sig, const hs_proposal_message & proposal){
+   bool qc_chain::evaluate_quorum(const extended_schedule& es, const boost::dynamic_bitset<>& finalizers, const fc::crypto::blslib::bls_signature& agg_sig, const hs_proposal_message& proposal) {
 
       bool first = true;
 
       if (positive_bits_count(finalizers) < _pacemaker->get_quorum_threshold()){
          return false;
       }
 
-      boost::dynamic_bitset fb(21, finalizers.value);
       fc::crypto::blslib::bls_public_key agg_key;
 
-      for (boost::dynamic_bitset<>::size_type i = 0; i < fb.size(); i++) {
-         if (fb[i] == 1){
+      for (boost::dynamic_bitset<>::size_type i = 0; i < finalizers.size(); i++) {
+         if (finalizers[i] == 1){
```

Comment:
```suggestion
         if (finalizers[i]){
```
Comparing with 1 is not quite right since operator[]() returns a `bool`